utils/maybe_async/README.md

```diff
@@ -0,0 +1,8 @@
+# Winter maybe-async
+This crate contains a `maybe_async` proc macro that abstracts away sync/async. It is heavily based on [`maybe-async`](https://github.com/fMeow/maybe-async-rs).
+
+
```

Comment:
Could we add a short explanation (with maybe a dummy example) of how this would work? For example, mention that we need to have `async` feature and how this feature affects the async/await code.

Reply:
Done